Documentation_displayed_on_the_portal/Share_Dataset_Instruction_Page.md

```diff
@@ -0,0 +1,222 @@
+
+- **_on Mac OS X_**
+
+Please visit the Install DataLad section of the [DataLad Handbook](http://handbook.datalad.org/en/latest/index.html) and follow the instruction for [Mac OS X install](http://handbook.datalad.org/en/latest/intro/installation.html#macos-osx).
+
+
+#### <a name="dataset_creation"></a> 2) Creation of a new DataLad dataset
+
+The first step to upload a dataset to CONP via DataLad requires the creation of a new DataLad dataset that will be tracked on GitHub.
+
+- Create a local DataLad directory:
+
+  ```
+  datalad create <new_dataset_name>
+  ```
+
+- Create a sibling for your dataset on GitHub. The command below will generate a sibling in your local space:
+
+  ```
+  cd <new_dataset_name>
+  datalad create-sibling-github <new_dataset_name>
+  ```
+  To inspect existing siblings, run `datalad siblings`
+	
+#### <a name="dataset_population"></a> 3) Population of the new dataset
+
+Choice of how to populate a new dataset will vary based on the special remote providing access to the data. The following procedure covers working with the [web](https://git-annex.branchable.com/tips/using_the_web_as_a_special_remote/) special remote. 
+
+**Dataset requirements:**
+
+- All datasets must include a `README.md` file in the root directory. That file will be displayed on the CONP portal page that will describe the dataset.
+- All datasets must include a `DATS.json` file as described in [the main documentation page](https://github.com/CONP-PCNO/conp-documentation/blob/master/CONP_DATS_fields.md). The content of this file will also be displayed on the CONP portal page that will describe the dataset. We provide a [DATS GUI editor]() ***(#TODO: ADD LINK TO EDITOR)*** for easy creation of this file. 
+- A logo can be added at the root of the dataset. This is optional, however, if there is a logo at the root of the dataset, it will be used on the CONP portal page that will describe the dataset.
+
+
+**How to track the different files of the dataset:**
+
+All files in the dataset must be added to the repository using one of the two commands below. Copying content from another location into your local copy of the repository without using those commands will not work.
+
+- For data files on FTP or HTTP servers, use the `web` remote to populate the data:
+
+	```
+	git annex addurl <URL_of_resource> --file <linkname>
+	```
+	*The `--file` switch is optional but recommended, because without it the default name for a link is built from the full URL of the resource and tends to be unwieldy and/or uninformative.
+	
+	*NB: Generating the link requires enough space on your local machine to store the large data file, as `git-annex` needs to download the file to generate checksums.
+
+- For metadata files (a.k.a. `README.md`, `DATS.json`, logo files), use `git` so that they are not annexed and readable by the CONP portal and users: 
```

Comment:
"a.k.a" ->"e.g.", I think; not impossible we may end up having more metadata files in future.

Also "not annexed and readable" is ambiguous, I suggest "not annexed, and therefore readable"

Reply:
ok